packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -119,16 +119,25 @@ const SYNC_FILTERS = [
 ];
 
 function PreviewWrapper( { item, onClick, ariaDescribedBy, children } ) {
-	if ( item.type === PATTERN_TYPES.theme ) {
-		return children;
-	}
 	return (
 		<button
 			className="page-patterns-preview-field__button"
 			type="button"
-			onClick={ onClick }
+			onClick={ ( event ) => {
+				if ( item.type === PATTERN_TYPES.theme ) {
+					event.preventDefault();
+				} else {
+					onClick( event );
+				}
+			} }
 			aria-label={ item.title }
 			aria-describedby={ ariaDescribedBy }
+			aria-disabled={ item.type === PATTERN_TYPES.theme }
+			title={
+				item.type === PATTERN_TYPES.theme
```

Comment:
Do we need to add this if it's disabled and we properly set the aria-disabled and onClick is noop?

Reply:
I removed the title attribute and changed the curser when the button is disabled.